mod/01_rating/spec/set_pattern/calculation_type_spec.rb

```diff
@@ -0,0 +1,21 @@
+# -*- encoding : utf-8 -*-
+
+describe Card::CalculationTypeSet do
+  it 'works' do
+    Card::Auth.as_bot do
+      Card[:rating].update_attributes! :name=>'Rating'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.